packages/create/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@readme/create",
+  "version": "0.0.1",
+  "description": "Sync your OpenAPI definition to ReadMe",
+  "bin": "../../bin/rdme openapi --github",
```

Comment:
I don't think this will work if you pull `@readme/create` down as it'll try to run `../../bin/rdme` on your machine.

Reply:
how is it any different than what's being done in this line? https://github.com/readmeio/rdme/blob/fc22c1833ae7ffa4c05ad470fda1a3d0938a5dfc/package.json#L11

Reply:
ohhh wait I think I know what you mean. I bet it's not even installing `rdme` to begin with. duh

Reply:
If we publish `@readme/create` to npm we only publish what's in `packages/create`, which won't include the `bin` directory.

Reply:
Can we use `npx` here at all?

```
"bin": "npx rdme openapi --github"
```

Reply:
jinx lol https://github.com/readmeio/rdme/pull/597/commits/a85690408b07d0faadbbb73036a05ccfdd5bc10d

Reply:
Or `@readme/create` can just have `rdme@latest` as a dependency and then we can call that from an `index.js` file like they do in `create-react-app`:

* https://github.com/facebook/create-react-app/blob/main/packages/create-react-app/package.json#L25
* https://github.com/facebook/create-react-app/blob/main/packages/create-react-app/index.js

Reply:
the commit I just pushed feels a little lighter than the CRA approach—are there any upsides besides the ability to write internal comments?

Reply:
Nah, if the commit you pushed worked I'm cool w/ that.

Reply:
the thing is I have no idea how to test it lol